pkg/chrootarchive/archive_unix.go

```diff
@@ -38,7 +43,17 @@ func untar() {
 		root = flag.Arg(1)
 	}
 
-	if root == "" {
+	// FreeBSD doesn't have proc/self, but we can handle it here
+	if root == "/proc/self/fd/4" {
+		// Take ownership to ensure it's closed; no need to leak
+		// this afterwards.
```

Comment:
(FWIW this is running a a dedicated process anyway, but I do prefer closing things explicitly, in case the code were refactored / reused in other contexts.)